src/pretix/plugins/statistics/views.py

```diff
@@ -162,4 +163,53 @@ def get_context_data(self, **kwargs):
 
         ctx['has_orders'] = self.request.event.orders.exists()
 
+        ctx['seats'] = {}
+
+        if not self.request.event.has_subevents or (ckey != "all" and subevent):
+            ev = subevent or self.request.event
+            if ev.seating_plan_id is not None:
+                seats_qs = ev.free_seats(sales_channel=None, include_blocked=True)
+                ctx['seats']['purchased_seats'] = len(ev.seats.annotate(has_op=Count('orderposition')))
+                ctx['seats']['blocked_seats'] = len(seats_qs.filter(blocked=True))
+                ctx['seats']['free_seats'] = len(seats_qs.filter(blocked=False))
+
+                seats_qs = seats_qs.select_related("product").values('product', 'blocked').annotate(count=Count('id'))\
+                    .order_by('product', 'blocked')
```

Comment:
Maybe

```suggestion
                    .order_by('product__category__position', 'product__position', 'product', 'blocked')
```
to keep the natural order of the products that we use everywhere?